content/en/docs/event-sources/gvisor/_index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: gVisor
+description: 'gVisor support in Falco'
+linktitle: gVisor
+weight: 80
+---
+
+Falco can work with **[gVisor](https://gvisor.dev/)**.
+
+gVisor, quoting the [official documentation](https://gvisor.dev/docs), is an application kernel that provides an **additional layer of isolation** between running applications and the host operating system. It delivers an additional security boundary for containers by **intercepting and monitoring workload runtime instructions in user space** before they can reach the underlying host.
+
+## How Falco and gVisor work together
+
+When running containers with gVisor, there are several components that interact with our workload:
+
+![](images/gvisor-01.png)
+
+The Sentry is the gVisor component that implements all the application kernel functionalities. In particular, from the Falco perspective, the Sentry abstracts the system call layer and manages almost every syscall an application can ever execute. In other words, the Sentry could be seen as an alternative to our drivers: it is in the right position to put together the information contained in the events that our eBPF probe or kernel module usually generates. So, how do we turn the Sentry into a *new driver* for Falco?
+
+The key observation here is that there is one Sentry process for each gVisor sandbox. If we want them to be able to communicate with Falco, we must set up a form of inter-process communication.
+
+We decided to use a UDS (Unix Domain Socket) to handle the communication between each Sentry and Falco. Falco is acting as a server, and it is responsible for setting up the socket and listening to connections. On the other hand, each Sentry process acts as a client, and it is configured to connect to the endpoint where Falco is listening.
+
+Whenever a syscall gets executed inside the sandboxed application, the Sentry will manage it as usual, plus it will send a message to Falco through the UDS. Messages are serialized through Protocol Buffers so that gVisor and Falco can communicate even if they are written in different programming languages.
+
+Once a message related to a syscall is received, Falco unpacks it and creates the corresponding event in a way that is consumable by our libraries. This way, it is possible to update necessary state information and trigger Falco rules whenever a match occurs!
```

Comment:
Could you summarize this part? While I agree that it is important to indicate that we connect to every configured gvisor sandboxes via a (configurable) unix domain socket I don't think the documentation page needs to explain the process that brought it to life in detail